h2/src/test/org/h2/test/TestAll.java

```diff
@@ -988,6 +989,7 @@ private void testUtils() {
         addTest(new TestUtils());
         addTest(new TestValueHashMap());
         addTest(new TestLocalResultFactory());
+        addTest(new TestSubqueryPerformanceOnLazyExecutionMode());
```

Comment:
This test should be invoked from `test()` or `testAdditional()`.

`testUtils()` is only for utilities that don't use connections and don't depend on test configuration.

Reply:
Fixed.